dipy/reconst/shm.py

```diff
@@ -1575,3 +1576,100 @@ def convert_sh_to_legacy(sh_coeffs, sh_basis, full_basis=False):
         raise ValueError("Invalid basis name.")
 
     return out_sh_coeffs
+
+
+def _setup_wigner_rotation(quant_order, alpha, beta, gamma):
+    """Returns an array of all possible wigner-D matrices for a range 
```

Comment:
- "Return" instead of "Returns" ( imperative mood)
- It should be one-line summary. we follow Numpy docstring standard: https://numpydoc.readthedocs.io/en/latest/format.html